doc/classes/ProjectSettings.xml

```diff
@@ -1709,6 +1709,32 @@
 			Uses a simplified method of generating PVS (potentially visible set) data. The results may not be accurate where more than one portal join adjacent rooms.
 			[b]Note:[/b] Generally you should only use this option if you encounter bugs when it is set to [code]false[/code], i.e. there are problems with the default method.
 		</member>
+		<member name="rendering/quality/blob_shadows/enable" type="bool" setter="" getter="" default="true">
+			If [code]true[/code], blob shadows will be rendered where present.
+			[b]Note:[/b] Blob shadows depend on a [BlobShadow] caster being present, and a [Light] that is set to cast blob shadows.
+		</member>
+		<member name="rendering/quality/blob_shadows/gamma" type="float" setter="" getter="" default="1.0">
+			A gamma function is applied in the shader, which can be used to determine the softness of blob shadows.
+		</member>
+		<member name="rendering/quality/blob_shadows/intensity" type="float" setter="" getter="" default="0.7">
+			Global setting for blob shadow intensity (darkness).
+			Used in conjunction with [member rendering/quality/blob_shadows/gamma] to determine shadow appearance.
+		</member>
+		<member name="rendering/quality/blob_shadows/max_capsules" type="int" setter="" getter="" default="4">
+			Blob shadow performance is tightly linked to the maximum casters (sphere and capsule).
+			It is advisable to keep this limit as low as possible, and make use of the [BlobFocus] node to ensure shadows appear in the gameplay area.
+			[b]Note:[/b] If you are only using sphere casters, you should set this value to zero for maximum performance.
+		</member>
+		<member name="rendering/quality/blob_shadows/max_spheres" type="int" setter="" getter="" default="4">
+			Blob shadow performance is tightly linked to the maximum casters (sphere and capsule).
+			It is advisable to keep this limit as low as possible, and make use of the [BlobFocus] node to ensure shadows appear in the gameplay area.
+			[b]Note:[/b] If you are only using capsule casters, you should set this value to zero for maximum performance.
+		</member>
```

Comment:
If I read the source code correctly, there's a hard limit of `MAX_CASTERS = 128` blobs that can be seen at a given time regardless of the project settings. This means that you could have up to 128 spheres *or* capsules (not both), or 64 spheres + 64 capsules at most.